### PR TITLE
Linking and Integration tasks

### PR DESCRIPTION
**Pull request by arronei**
- Linking fixes
- Normative/non-normative link fixes
- Update coords parsing

### CODE REVIEW

**Comment by arronei** [comment]
Part of #364

File: sections/element-interfaces.include [hunk 1]
```diff
@@ -370,6 +374,10 @@
      </td><td> {{HTMLSourceElement}} : {{HTMLElement}}
 
     </td></tr><tr>
+     <td> <a element lt="picture source"><code>source</code></a>
+     </td><td> <a href='#Picture-HTMLSourceElement'><code>HTMLSourceElement</code></a> : {{HTMLElement}}
```
Comment (travisleithead) [inline sections/element-interfaces.include:378]: {{HTMLSourceElement}} doesn't work?

Comment (travisleithead) [inline sections/element-interfaces.include:378]: ...and <{source}>

File: sections/elements.include [hunk 2]
```diff
@@ -1263,8 +1263,8 @@
          <{source/srcset}>;
          <{source/sizes}>;
          <{source/media}>;
-         <a element-attr lt='picture source type'><code>type</code></a></td>
-     <td><a href='#Picture-HTMLSourceElement'><code class='idl'>HTMLSourceElement</code></a></td>
+         <{source/type}></td>
+     <td><a href='#Picture-HTMLSourceElement'><code>HTMLSourceElement</code></a></td>
```
Comment (travisleithead) [inline sections/elements.include:1267]: {{HTMLSourceElement}} ?

**Comment by travisleithead** [comment]
Had a few comments, but not blocking (and can be picked up in a subsequent commit). This is otherwise a no-brainer.

**Pull request merged**